pkgs/applications/misc/rm-improved/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, rustPlatform, lib }:
```

```suggestion
{ fetchFromGitHub, rustPlatform, lib }:
```
Per `nixpkgs-hammer`